Sources/TensorFlow/Layers/Recurrent.swift

```diff
@@ -444,12 +461,98 @@ public struct RecurrentLayer<Cell: RecurrentLayerCell>: Layer {
   }
 }
 
+public protocol Mergeable: Differentiable, AdditiveArithmetic {
+  @differentiable
+  static func +(lhs: Self, rhs: Self) -> Self
+}
+
+@differentiable
+public func sum<TimeStepOutput: Mergeable>(
```

Comment:
This `sum` function is generic over `Mergeable`, but nothing should be specifically named `TimeStepOutput`.
```suggestion
public func sum<T: Mergeable>(
```